ta/pkcs11/src/processing_asymm.c

```diff
@@ -78,6 +100,11 @@ pkcs2tee_algorithm(uint32_t *tee_id, uint32_t *tee_hash_id,
 		break;
 	}
 
+	if (*tee_id == TEE_ALG_RSAES_PKCS1_V1_5 &&
```

Comment:
I think this deserves an inline comment, something like
```c
/*
 * PKCS11 uses unique CKM_RSA_PKCS identifier for both ciphering and authentication
 * whereas GPD TEE expects TEE_ALG_RSAES_PKCS1_V1_5 for ciphering and
 * TEE_ALG_RSASSA_PKCS1_V1_5 for authentication.
 */

Reply:
Adding similar text